### PR TITLE
VIR-87 Position name frontend

### PR DESCRIPTION
**Pull request by nniemi**
Added position name field to creating new position
Possible to select from a list of options and write your own

### CODE REVIEW

File: backend/Virkarekisteri/src/Functions/PositionName/GetPositionNames.cs [hunk 1]
```diff
@@ -0,0 +1,38 @@
+﻿using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Azure.Functions.Worker;
+using Microsoft.Extensions.Logging;
+using Virkarekisteri.Repositories;
+
+namespace Virkarekisteri.Functions.PositionName
+{
+    public class GetPositionNames
```
Comment (joonasyr) [inline backend/Virkarekisteri/src/Functions/PositionName/GetPositionNames.cs]: Could use a simpler constructor here, similar to CreatePosition function. @0x464e mentioned this before in another PR

Comment (nniemi) [reply backend/Virkarekisteri/src/Functions/PositionName/GetPositionNames.cs]: done

File: frontend/src/components/Modal/CreateVirkaModal.tsx [hunk 2]
```diff
@@ -28,8 +31,21 @@ interface CreateVirkaModalProps {
...
+    }
+  }, [dispatch, open]);
 
   const onSubmit = async (values: Partial<Position>) => {
+    console.log(values);
+    const positionNameObj: PositionName = {
+      name: values.positionName?.toString() || '',
+    };
     try {
       const positionData: Position = {
```
Comment (joonasyr) [inline frontend/src/components/Modal/CreateVirkaModal.tsx:49]: Could maybe add the OrgTreeId to Position model to match backend and prevent crashes, but can also be done in the next ticket

**Review by joonasyr — DISMISSED** [review]
Looks good!

**Review by joonasyr — APPROVED** [review]

**Pull request merged**